### ISSUE TITLE
Does MCA support PVA? 

### ISSUE THREAD

**Issue by jeonghanlee** [EPICSTools/mca#2]
@kasemir , I am wondering about MCA support for PVA. Does it ?

**Comment by kasemir** [comment]
No, it's Channel Access.

For Matlab, though, you can call Java libs.
The new Java PVA client, https://github.com/ControlSystemStudio/phoebus/tree/master/core/pva, is specifically avoiding Java 9+ code so that it still compiled with Java 8 and can thus be called from Matlab.
SLAC has been using that.

**Comment by kasemir** [comment]
Hugo Slepicka @ SLAC has been trying that.

**Comment by jeonghanlee** [comment]
> Hugo Slepicka @ SLAC has been trying that.

"That" is the new JAVA PVA client?

**Comment by jeonghanlee** [comment]
I have one more question.

* what kind of relation between the new JAVA PVA client with https://github.com/epics-base/pvaClientJava/tree/master/src/org/epics/pvaClient? 
* Is this the why you call it "new JAVA PVA client"?

**Comment by kasemir** [comment]
It's a new/separate development for Java, similar to pvxs for C++.
The original libs tried to have the same API for Java and C++, so they could not take advantage of the respective language, instead getting the worst of both.
The new developments also just have 'container' type value classes, not one copy of classes just to describe, and one to actually hold data, since the only time where you need info-without-data is the 'pvinfo' command, where you might as well get a structure with all NaN/0/empty.
In short, the Java lib is must smaller, works for all the CS-Studio use cases, and shows same or better performance as far as CPU and memory are concerned for 'NTNDArray' images.
I've had presentations at EPICS developer's meetings.

**Comment by jeonghanlee** [comment]
@kasemir  thank you for kind explanation. Sorry about not following EPICS meeting presentations.
